lib/db/models/Order.ts

```diff
@@ -0,0 +1,48 @@
+import Sequelize from 'sequelize';
+import { db } from '../../types';
+
+export default (sequelize: Sequelize.Sequelize, DataTypes: Sequelize.DataTypes) => {
+  const attributes: db.SequelizeAttributes<db.OrderAttributes> = {
+    id: { type: DataTypes.STRING, primaryKey: true, allowNull: false },
+    peerNodeId: { type: DataTypes.STRING },
+    localId: { type: DataTypes.STRING },
+    initialQuantity: { type: DataTypes.DECIMAL(8), allowNull: true },
+    pairId: { type: DataTypes.STRING, allowNull: false },
+    price: { type: DataTypes.DECIMAL(8), allowNull: false },
```

Comment:
Yeah good point, I believe `DECIMAL` types can handle that but I also don't think it's very graceful to store zero price and max number prices in the database. I think it might be nicest to make `price` nullable and leave it as null for market orders - as effectively they don't have a price. We have the `isBuy` property to indicate the direction of the order, so I don't think the price provides any meaningful information for market orders. Does that make sense?

Reply:
I can handle that change in a future PR, for the time being we aren't saving any market orders to the database.